ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_statistical.py

```diff
@@ -182,3 +182,50 @@ def test_unravel_index(
         indices=np.asarray(x[0], dtype=input_dtype[0]),
         shape=shape,
     )
+
+    
+# nanquantile
+
+@handle_test(
+    fn_tree="functional.experimental.nanquantile",
+    dtype_x_axis=statistical_dtype_values(function="nanquantile"),
+    q=st.floats(min_value=0, max_value=1),
+    keepdims=st.booleans(),
+    interpolation=st.data(),
+)
+def test_nanquantile(
+    *,
+    q,
+    dtype_x_axis,
+    keepdims,
+    interpolation,
+    ground_truth_backend,
+    num_positional_args,
+    as_variable,
+    with_out,
+    native_array,
+    container_flags,
+    instance_method,
+    backend_fw,
+    fn_name,
+    on_device,
+):
+    input_dtype, x, axis = dtype_x_axis
```

Comment:
for unpacking the values, this should be:
`input_dtype, x, axis, interpolation, q  = dtype_x_axis`